tests/unit/test_athena.py

```diff
@@ -19,10 +20,20 @@
     get_df_list,
     get_df_txt,
     get_time_str_with_random_suffix,
+    pandas_equals,
 )
 
+if wr.engine.get() == EngineEnum.RAY and wr.memory_format.get() == MemoryFormatEnum.MODIN:
+    import modin.pandas as pd
+else:
+    import pandas as pd
+
 logging.getLogger("awswrangler").setLevel(logging.DEBUG)
 
+pytestmark = pytest.mark.distributed
+
+is_ray_modin = wr.engine.get() == EngineEnum.RAY and wr.memory_format.get() == MemoryFormatEnum.MODIN
```

Comment:
Nitpick, could move this to line 25 and then use it for the modin vs pandas conditional:
```
is_ray_modin = wr.engine.get() == EngineEnum.RAY and wr.memory_format.get() == MemoryFormatEnum.MODIN
if is_ray_modin:
    import modin.pandas as pd
else:
    import pandas as pd
```

As a side note, we make this check and import in so many places that we should perhaps put them in our tests fixtures/utils?

Reply:
Great idea, I like it.